### PR TITLE
Sprint 67 Work

### PR DESCRIPTION
**Pull request by JPrause**
Update for changelog

### CODE REVIEW

**Comment by Fryguy** [comment]
@mfeifer Please review

**Comment by Fryguy** [comment]
We don't really have milestones here as the release cycle of the container is independent of ManageIQ.  I'm wondering if we should change the section title to just "Unreleased" a la http://keepachangelog.com

**Comment by mfeifer** [comment]
@JPrause agree with Jason, should be Unreleased as of....similar to https://github.com/ManageIQ/manageiq/blob/master/CHANGELOG.md

Otherwise, looks good.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+## Unrelease - as of Sprint 67 ending 2017-08-21
```
Comment (Fryguy) [inline CHANGELOG.md]: @JPrause Typo `Unrelease` => `Unreleased`.

Comment (Fryguy) [reply CHANGELOG.md]: You also need the Added/Updated/Fixed header as per keepachangelog.com

**Comment by JPrause** [comment]
@miq-bot add_label documentation

**Comment by JPrause** [comment]
@Fryguy updated as per our conversation. Please merge.

**Pull request merged**